LinkBase/LinkBase/UtilityFunctions.swift

```diff
@@ -46,3 +46,27 @@ func getDateFromPriorWeek(currentDate: Date, numWeeks: Int) -> Date {
  
     return newDate
 }
+
+
+// this method removes the strings from the first and last name properties in Parse
+func getUserFirstName(user: User) -> String {
+   
+    let _firstname = user["firstname"]
```

Comment:
In certain circumstances, that can in fact happen.  With the merge of PR #23, that should no longer be the case.  Basically, this occurred because the initial project had updated a field by using the add method in Parse.  I did not realize it at the time, but when you use this method, Parse treats the field as an array (and even if you try to force downcast as a string, Parse ignores this).  And to add further note on what to look out for, the Parse schema keeps a copy of the old field as an array.  So in order to fix this, it requires updating the Parse schema, and deleting any objects with the old schema structure.

Above is just to explain in case you come across this somewhere else.  The issue should now be resolved in the project.